src/main/java/graphql/nadel/NadelExecutionHints.java

```diff
@@ -0,0 +1,33 @@
+package graphql.nadel;
+
+public class NadelExecutionHints {
```

Comment:
I'm not sure whether this should be tagged with `@PublicApi`

Reply:
Do it